### PR TITLE
improvement: improve model

### PR DESCRIPTION
**Pull request by admantiumblack**
1. change ollama to chatgpt to improve latency.
2. tweak prompt to enable the model to ask questions about user profile.

### CODE REVIEW

File: agents/blueprints/financial_planner_agent.py [hunk 1]
```diff
@@ -15,10 +15,9 @@ def create_prompt_generator(pipeline: Pipeline, template_path: str, agent_name:
 
     pipeline.add_component(
         agent_name,
-        OllamaGenerator(
-            model=os.getenv("LLM_MODEL", "gemma"),
-            url=str(os.environ["LLM_URL"]),
-            timeout=int(os.getenv("LLM_TIMEOUT_SECONDS", 10 * 60)),
+        OpenAIGenerator(
+            model=os.getenv("LLM_MODEL", "gpt-3.5-turbo"),
+            # timeout=int(os.getenv("LLM_TIMEOUT_SECONDS", 10 * 60)),
```
Comment (elvanselvano) [inline agents/blueprints/financial_planner_agent.py]: Is no timeout expected?

File: agents/blueprints/financial_planner_agent.py [hunk 2]
```diff
@@ -29,14 +28,14 @@ def financial_planner(pipeline_kwargs: dict = {}):
     # code is designed this way due to approaching deadline
     # TODO: refactor codes to make it more reusable
     planner = Pipeline(**pipeline_kwargs)
-    planner.add_component("profile", Multiplexer(str))
+    # planner.add_component("profile", Multiplexer(str))
```
Comment (elvanselvano) [inline agents/blueprints/financial_planner_agent.py]: Avoid commented codes

File: agents/blueprints/financial_planner_agent.py [hunk 3]
```diff
@@ -29,14 +28,14 @@ def financial_planner(pipeline_kwargs: dict = {}):
...
     planner.add_component("chat", Multiplexer(list[dict[str]]))
 
     agents = ["risk_analyst", "investment_consultant", "financial_planner"]
     for agent in agents:
         create_prompt_generator(planner, f"agents/templates/{agent}.txt", agent)
         planner.connect("chat.value", f"{agent}_template_builder.chat_log")
-        planner.connect("profile.value", f"{agent}_template_builder.profile")
+        # planner.connect("profile.value", f"{agent}_template_builder.profile")
```
Comment (elvanselvano) [inline agents/blueprints/financial_planner_agent.py]: Avoid commented codes

File: agents/blueprints/profile_extractor_agent.py [hunk 4]
```diff
@@ -15,10 +15,9 @@ def profile_extractor(pipeline_kwargs: dict = {}):
     extractor.add_component("builder", PromptBuilder(template))
     extractor.add_component(
         "generator",
-        OllamaGenerator(
-            model=os.getenv("LLM_MODEL", "gemma"),
-            url=str(os.environ["LLM_URL"]),
-            timeout=int(os.getenv("LLM_TIMEOUT_SECONDS", 10 * 60)),
+        OpenAIGenerator(
+            model=os.getenv("LLM_MODEL", "gpt-3.5-turbo"),
+            # timeout=int(os.getenv("LLM_TIMEOUT_SECONDS", 10 * 60)),
```
Comment (elvanselvano) [inline agents/blueprints/profile_extractor_agent.py]: Is no timeout expected?

Comment (admantiumblack) [reply agents/blueprints/profile_extractor_agent.py]: No, openaigenerator does not accept timeout

File: agents/blueprints/financial_planner_agent.py (hunk 1 above)
Comment (admantiumblack) [reply agents/blueprints/financial_planner_agent.py]: No, openaigenerator does not accept timeout

File: agents/blueprints/financial_planner_agent.py (hunk 2 above)
Comment (admantiumblack) [reply agents/blueprints/financial_planner_agent.py]: Fixed

File: agents/blueprints/financial_planner_agent.py (hunk 3 above)
Comment (admantiumblack) [reply agents/blueprints/financial_planner_agent.py]: Fixed

File: .env.example [hunk 5]
```diff
@@ -1,3 +1,6 @@
-LLM_MODEL="gemma"
+LLM_MODEL="gpt-3.5-turbo-0125"
 LLM_URL="http://ollama:11434/api/generate"
-LLM_TIMEOUT_SECONDS=300 #second
\ No newline at end of file
+OPENAI_API_KEY=
+LLM_TIMEOUT_SECONDS=600 #second
```
Comment (elvanselvano71) [inline .env.example]: Please validate unused env

**Review by elvanselvano71 — APPROVED** [review]

**Pull request merged**